packages/lighthouse-service/src/property-manager/helpers.ts

```diff
@@ -46,3 +46,26 @@ export const populateMongooseDocWithUser = async (
   property.updatedBy = user;
   return property;
 };
+
+/**
+ * Helper function to create LH Project
+ * @param {PropertyModel} $project: input project document
+ * @returns {Object}: Mutated document as object
+ */
+export const createLHProject = ( project: any ) => {
```

Comment:
After something property manager will be completely removed. so this needs to be moved to audit manager

Reply:
As discussed this will be taken care when the property manager will get removed.

Reply:
Ok